edx-platform/pearson-vue-theme/lms/templates/ccx/enrollment.html

```diff
@@ -0,0 +1,123 @@
+<%page expression_filter="h"/>
+<%!
+from django.utils.translation import ugettext as _
+from django.utils.translation import pgettext
+from openedx.core.djangolib.markup import HTML, Text
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+%>
+
+<%
+    allow_revoke_access = configuration_helpers.get_value('ALLOW_CCX_REVOKE_ACCESS', True)
```

Comment:
Shouldn't the default be False? If the import fails, it is still True and disables the revoke access. In this case I think we want to do it just for license managed courses. Perhaps the condition should be `if allow_revoke_access && is_license_managed`.

Reply:
@elvinquero, No, the default should be True, and we only check if the CCX is license managed when we have the feature turned on. Does that make sense?

Reply:
@ivanvgh So, under these changes, no matter if the ALLOW_CCX_REVOKE_ACCESS is turned on or not. It will disable the enrollment for all Sites? I don't think this should be the case.

Reply:
@elvinquero, allow_revoke_access means that the button is shown, so by default is true, and it only affects the VUE theme.